src/style/history.scss

```diff
@@ -1,6 +1,6 @@
 #past-swaps {
```

Comment:
Also, this is in the wrong file. The id is defined in `SwapList` which we also use in `Refund`